pygfx/renderers/wgpu/engine/pipeline.py

```diff
@@ -771,6 +776,10 @@ def _compile_shader(self, pass_index, env):
         env_kwargs = env.get_shader_kwargs(env_bind_group_index)
         shader_kwargs = blender_kwargs.copy()
         shader_kwargs.update(env_kwargs)
+        if material is not None:
+            shader_kwargs["write_pick"] = (
+                shader_kwargs["write_pick"] and material.pick_write
+            )
```

Comment:
I think this is a rare opportunity to use the `&=` operator:

```py
>>> write_pick = True
>>> write_pick &= True
>>> write_pick
True
>>> write_pick &= False
>>> write_pick
False
```

Reply:
> rare opportunity

so rare i dont even think about it ^_^